File: Java/Warmup-1/hasTeen.java

public class warmup1 {

	// this function return true if the numbers given are in range 13-19 inclusive
	public boolean hasTeen(int a, int b, int c) {
		
		boolean bool = false;
		
		if ((a >= 13 && a <= 19) || (b >= 13 && b <= 19) || (c >= 13 && c <= 19))
			bool = true;
		
		return bool;
		
	}
	
	public static void main(String[] args) {
		
		warmup1 o = new warmup1();
		
		System.out.println(o.hasTeen(13,20,10));
		System.out.println(o.hasTeen(20,19,10));
		System.out.println(o.hasTeen(20,10,13));
		
	}
	
}

File: Python/Warmup-1/has_teen.py

# this function return true if the given int values are in range 13-19 inclusive
def has_teen(a,b,c):
	
	b = False

	if (a >= 13 or a <= 19) and (b >= 13 or b <= 19) and (c >= 13 or c <= 19):
		b = True

	return b

print(has_teen(13,20,10))
print(has_teen(20,19,10))
print(has_teen(10,10,13))
